query/query.go

```diff
@@ -2286,6 +2296,14 @@ func ProcessGraph(ctx context.Context, sg, parent *SubGraph, rch chan error) {
 		}
 	}
 
+	// We apply Every _after_ pagination, so it refers to the respective page
+	if sg.Params.Every > 1 {
+		if err = sg.applyEvery(ctx); err != nil {
```

Comment:
we should use := here to reduce the scope of err

Reply:
Many other place do not limit the scope of err, so I have reduced the scope in all those other places as well: eebba60020a5ce17be038a23bbb645ac57b5c896

Reply:
Thanks